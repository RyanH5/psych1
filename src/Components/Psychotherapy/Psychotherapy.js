import React from 'react';

export const Psychotherapy = () => {
  return (
    <div>
      <h1>Psychotherapy</h1>
      <p>
      Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.
      </p>
      <p>
      I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.
      </p>
    </div>
  )
}