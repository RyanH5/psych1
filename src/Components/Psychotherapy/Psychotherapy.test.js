import React from 'react';
import { shallow } from 'enzyme';
import { Psychotherapy } from './Psychotherapy';

describe('Psychotherapy', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Psychotherapy />)

    expect(wrapper).toMatchSnapshot();
  });
});