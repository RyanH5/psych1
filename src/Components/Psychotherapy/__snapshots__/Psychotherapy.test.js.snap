// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Psychotherapy should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Psychotherapy
</h1>,
        <p>
          Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.
</p>,
        <p>
          I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Psychotherapy",
        },
        "ref": null,
        "rendered": "Psychotherapy",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.",
        },
        "ref": null,
        "rendered": "Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.",
        },
        "ref": null,
        "rendered": "I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Psychotherapy
</h1>,
          <p>
            Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.
</p>,
          <p>
            I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Psychotherapy",
          },
          "ref": null,
          "rendered": "Psychotherapy",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.",
          },
          "ref": null,
          "rendered": "Psychotherapy can be helpful when we are struggling with one or more areas of our life. We all have patterns that can continue to contribute to our confusion and pain and we have patterns that contribute to our happiness and wellbeing. My belief is that we are always changing and there is opportunity for growth. We all have the internal resources that can help us interrupt the unhealthy patterns and drop our old ways of staying stuck.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.",
          },
          "ref": null,
          "rendered": "I believe we have physical, cognitive and emotional parts of our self. When they are in balance we can navigate the ongoing changes and enjoy our life. When one or more of these areas are out of balance, we often find ourselves in distress and unable to enjoy life. Psychotherapy can be helpful with developing skills in interrupting and changing old habits and developing new patterns so we can participate in and enjoy life to the fullest. My approach in psychotherapy is to help people develop these skills as quickly as possible.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
